test/integration/02-user-wallet/02-send-lightning.spec.ts

```diff
@@ -1352,6 +1352,120 @@ describe("UserWallet - Lightning Pay", () => {
   })
 })
 
+describe("Delete payments from Lnd - Lightning Pay", () => {
+  it("deletes payment", async () => {
+    const { request, secret, id } = await createInvoice({ lnd: lndOutside1 })
+    const paymentHash = id as PaymentHash
+    const revealedPreImage = secret as RevealedPreImage
+
+    // Test payment is successful
+    const paymentResult = await Wallets.payNoAmountInvoiceByWalletId({
+      paymentRequest: request as EncodedPaymentRequest,
+      memo: null,
+      amount: amountInvoice,
+      senderWalletId: walletIdB,
+      senderAccount: accountB,
+      logger: baseLogger,
+    })
+    if (paymentResult instanceof Error) throw paymentResult
+    expect(paymentResult).toBe(PaymentSendStatus.Success)
+
+    const lndService = LndService()
+    if (lndService instanceof Error) return lndService
+
+    // Confirm payment exists in lnd
+    const retrievedPayment = await lndService.lookupPayment({ paymentHash })
+    expect(retrievedPayment).not.toBeInstanceOf(Error)
+    if (retrievedPayment instanceof Error) return retrievedPayment
+    expect(retrievedPayment.status).toBe(PaymentStatus.Settled)
+    if (retrievedPayment.status !== PaymentStatus.Settled) return
+    expect(retrievedPayment.confirmedDetails?.revealedPreImage).toBe(revealedPreImage)
+
+    // Delete payment
+    const deleted = await lndService.deletePaymentByHash({ paymentHash })
+    expect(deleted).not.toBeInstanceOf(Error)
+
+    // Check that payment no longer exists
+    const retrievedDeletedPayment = await lndService.lookupPayment({ paymentHash })
+    expect(retrievedDeletedPayment).toBeInstanceOf(PaymentNotFoundError)
+
+    // Check that deleting missing payment doesn't return error
+    const deletedAttempt = await lndService.deletePaymentByHash({ paymentHash })
+    expect(deletedAttempt).not.toBeInstanceOf(Error)
+  })
+
+  it("runs delete-payment cronjob", async () => {
```

Comment:
this test should be in cron path (test)

Reply:
Done https://github.com/GaloyMoney/galoy/pull/1193/commits/2a33a6323a15a564912bdde4cb0948e16a1d4867